src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarDetails.jest.tsx

```diff
@@ -265,7 +265,7 @@ describe("ArtworkSidebarDetails", () => {
         }),
       })
 
-      expect(screen.queryByText(/Curators' Pick/)).toBeInTheDocument()
+      expect(screen.queryByText(/Curators’ Pick/)).toBeInTheDocument()
```

Comment:
Oh. I assume this change should also be done on mobile, right?

Reply:
:ceiling-cat: This is already there in Eigen :)